### PR TITLE
Try removing check_includes = false in //brave/farbling

### PR DESCRIPTION
**Pull request by sangwoo108**
Resolves https://github.com/brave/brave-browser/issues/37200

## Submitter Checklist:

- [ ] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: browser/farbling/brave_navigator_languages_farbling_browsertest.cc [hunk 1]
```diff
@@ -33,7 +33,10 @@
 #include "content/public/test/content_mock_cert_verifier.h"
 #include "net/dns/mock_host_resolver.h"
 #include "third_party/blink/public/web/web_local_frame.h"
-#include "third_party/blink/renderer/core/frame/navigator_language.h"
+
+// TODO(sko) Depending on core directly isn't allowed by the visibility of the
+// target. it might be better to export it through public/ layer.
+#include "third_party/blink/renderer/core/frame/navigator_language.h"  // nogncheck
```
Comment (goodov) [inline browser/farbling/brave_navigator_languages_farbling_browsertest.cc:39]: do we really need this file in this test?

Comment (sangwoo108) [reply browser/farbling/brave_navigator_languages_farbling_browsertest.cc:39]: I have no idea, I just guarded. Let me try again

Comment (sangwoo108) [reply browser/farbling/brave_navigator_languages_farbling_browsertest.cc]: Yeah, you were right. Build passed without this include. Updated!

**Comment by sangwoo108** [comment]
@goodov @SergeyZhukovsky Would you mind taking a look? Thanks!

**Review by goodov — APPROVED** [review]

File: browser/extensions/BUILD.gn [hunk 2]
```diff
@@ -160,3 +160,26 @@ group("resources") {
     ]
   }
 }
+
+source_set("test_support") {
+  testonly = true
+
+  sources = [
+    "//brave/browser/extensions/brave_base_local_data_files_browsertest.cc",
+    "//brave/browser/extensions/brave_base_local_data_files_browsertest.h",
```
Comment (goodov) [inline browser/extensions/BUILD.gn:169]: nit: these should be just `"brave_base_local_data_files_browsertest.cc/h"`

**Review by SergeyZhukovsky — APPROVED** [review]
android tests ++
